philadelphia/src/test/java/org/jvirtanen/philadelphia/FIXSessionTest.java

```diff
@@ -0,0 +1,20 @@
+package org.jvirtanen.philadelphia;
+
+import org.junit.Test;
+
+/**
+ * Created on: 03/24/2016
+ *
+ * @author Alexander Aleshin
+ * @version %I%, %G%
+ */
+public class FIXSessionTest {
```

Comment:
Could you rename this class to `FIXConfigTest` as this is primarily testing the configuration? The actual session tests are in `FIXInitiatorTest`.

Also, could you please remove the automatically generated comment above as such comments are not present in the other test classes either?